charter.md

```diff
@@ -53,15 +53,207 @@ The following responsibilities belong to the Steering Committe, but may be deleg
+on decisions within the body's purview.
+
+These votes may proceed on agreed-upon platforms by the committee, such as:
+
+* a pull request
+* an issue
+* a Steering Committee [meeting](#meetings)
+* a mailing list
+
+Unless otherwise specified by a process, the requirement for passing a vote is
+a **_two-thirds supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 3 = 2.333...
+* 2.333... * 2 = 4.666...
+* Round up to the nearest whole number (5)
+* 5 members in attendance would be required to vote during a meeting
+
+### Abstention
+
+For any self-elected reason, members of the committee may decide to abstain
+from a vote.
+
+Abstaining members will only be considered as contributing to quorum, in the
+event that a vote is called in a meeting.
+
+## Meetings
+
+Steering Committee members are generally expected to attend every meeting. We
 use the following guidelines to determine whether we have reached quorum and
 are able to proceed with a meeting.
 
-* Round quorum up if total number of members is odd
-* Quorum to meet is 1/2 members attending (eg: 6/12, 6/11, 4/7)
-* Quorum to vote in a meeting is 2/3 members attending (eg: 8/12, 8/11, 5/7)
+### Quorum
+
+Quorum **to meet** is a **_majority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 2 = 3.5
+* Round up to the nearest whole number (4)
+* 4 members in attendance would be required to meet
+
+Quorum **to vote in a meeting** is a **_two-thirds supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 3 = 2.333...
+* 2.333... * 2 = 4.666...
+* Round up to the nearest whole number (5)
+* 5 members in attendance would be required to vote during a meeting
+
+## Elections
+
+Every year, the Steering Committee holds a general election for open seats with
+[SIG Contributor Experience][sig-contribex] owning the process.
+
+SIG Contributor Experience may delegate this responsibility to a subproject.
+
+Our [election policy document][general-elections] covers the details for how
+this works.
+
+Policy and operations of special elections initiated by the Steering Committee
+for other reasons (as yet to be determined) may be handled by some agreed-upon
+[process][changes] by the committee.
+
+[general-elections]: /elections.md
+[sig-contribex]: https://git.k8s.io/community/sig-contributor-experience
+
+### Vacancies
+
+In the event of a resignation, removal, or other loss of an elected Steering 
+Committee member, the candidate with the next most votes from the previous 
+election will be offered the seat. This process will continue until the seat is 
+filled. 
+
+In case this fails to fill the seat, a special election for that position will 
+be held as soon as possible. [Eligible voters](https://git.k8s.io/steering/elections.md#eligibility-for-voting) from the most recent election 
```

Comment:
Add a time limit, just leave the seat vacant if there is only a month until the next regular election. Personally I'd go as high as six months given how expensive elections are. But it would be silly to have a special election for a period of time when the board won't even meet.

If the board is missing enough people that it can't make the voting quorum then there really does need to be an emergency special election.

Reply:
I tend to agree with this. Perhaps an emeritus can fill the vacant spot temporarily in the example of a vacancy of a months or more (but not more than say 6 months)

Reply:
I don't think I would do that as it could select someone the community deliberately voted out.

I think the board should work fine with 1 unfilled seat, it just makes passing things slightly harder. With 2 unfilled seats everything has to be unanimous, which is iffy. With 3 unfilled seats they can't do anything at all.

Reply:
Very good points, thank you